### ISSUE TITLE
Tests last forever in any Kotlin track exercise

### ISSUE THREAD

**Issue by mnajborowski** [exercism/exercism#5844]
So far since the v3 release I've submited three Kotlin track exercises:
- Scrabble Score
- Matrix
- Difference Of Squares

I've submitted all of them through the CLI (because online editor is not working for me, which I'll describe in another issue). Tests pass locally for all of them. For the first two, both on first iteration, the online test runner failed immediately with the same error message as in #5657. After #5657 has been closed, I submitted the solution again, but now the tests just never end (also for the last exercise which I've just submitted for the first time). I've waited for 2h and still nothing.

Steps to reproduce:
1. Download the exercise locally through the CLI
2. Implement the solution
3. Run tests locally (all pass)
4. Submit solution through the CLI
5. Tests never end

Expected behavior:
Tests should finish shortly after solution submit (not expecting them to finish in 5-20 sec like the message states, but certainly before 2h of wait)

**Comment by ErikSchierboom** [comment]
As Kotlin doesn't yet have a functioning test runner, CLI submissions should not even _be_ tested (because they can't). See https://github.com/exercism/exercism/issues/5845#issuecomment-916089029 for a more elaborate description.

**Comment by ErikSchierboom** [comment]
Closed by https://github.com/exercism/kotlin-test-runner/pull/34
